Header header
string[] joint_names 
float64[] positions

# set to -1 for maximum 
float64[] velocities
float64[] accelerations
float64[] max_currents